projects/BFDR.Messaging/BirthRecordAcknowledgementMessage.cs

```diff
@@ -18,6 +18,7 @@ public BirthRecordAcknowledgementMessage(BirthRecordBaseMessage messageToAck) :
             this.StateAuxiliaryId = messageToAck?.StateAuxiliaryId;
             this.JurisdictionId = messageToAck?.JurisdictionId;
             this.BirthYear = messageToAck?.BirthYear;
+            this.PayloadVersionId = messageToAck?.PayloadVersionId;
```

Comment:
I think all of the message types should set the PayloadVersionId to $"{GeneratedCustomProperty.Value}" The PayloadVersionId tells us which IG was used to construct the acknowledgement, coded response etc. not what IG the original submission message was based on. 

If for example the submission message was BFDR STU 2.0 but the library is based of a slightly updated IG BFDR STU 2.1 then, the new ack message should set the payload version id to 2.1 for accuracy.